web/src/utils/cn.ts

```diff
@@ -0,0 +1,6 @@
+import { ClassValue, clsx } from "clsx"
```

Comment:
Could we rename this file to `classname` or something descriptive? I don't mind the function being short, but it's hard to grok why we're importing it with a short name

Reply:
Gotit! Definitely haha